ci-operator/step-registry/baremetal/lab/pre/reserve-nodes/baremetal-lab-pre-reserve-nodes-commands.sh

```diff
@@ -38,6 +38,7 @@ set +o allexport
 
 # shellcheck disable=SC2174
 mkdir -m 755 -p {/var/builds,/opt/tftpboot,/opt/html}/${BUILD_ID}
+mkdir -m 777 -p /opt/nfs/${BUILD_ID}
```

Comment:
Does this host also need firewalld setting in order for clients to mount?

Reply:
I set it to be a persistent rule in the `internal` firewalld zone of the current auxiliary host.

When the definitive implementation of it will be available, we will version that config under the github.com/openshift-qe/baremetal-qe-infra repo.